Can't compile using Clang 14

Client version: x.x.x
Server OS: Linux
Cuberite Commit id: 2e23b7cfba344bd24ae202b26ff280a57748570d


### Expected behavior
Compiles

### Actual behavior
```
cuberite/src/Simulator/IncrementalRedstoneSimulator/RedstoneSimulatorChunkData.h:71:15: error: no member named 'exchange' in namespace 'std'; did you mean '__exchange'?
                return std::exchange(Result->second, Power);
                       ~~~~~^~~~~~~~
                            __exchange
/usr/bin/../lib/gcc/x86_64-redhat-linux/12/../../../../include/c++/12/bits/move.h:157:5: note: '__exchange' declared here
    __exchange(_Tp& __obj, _Up&& __new_val)
```

### Steps to reproduce the behavior
Compile using Clang 14

--- a/src/Globals.h
+++ b/src/Globals.h
@@ -136,6 +136,7 @@
 #include <type_traits>
 #include <unordered_map>
 #include <unordered_set>
+#include <utility>
 #include <vector>
 #include <variant>
 

